whois-query/src/main/java/net/ripe/db/whois/query/pipeline/ExceptionHandler.java

```diff
@@ -43,6 +44,14 @@ public void exceptionCaught(final ChannelHandlerContext ctx, final ExceptionEven
             handleException(channel, Collections.singletonList(QueryMessages.inputTooLong()), QueryCompletionInfo.EXCEPTION);
         } else if (cause instanceof IOException) {
             handleException(channel, Collections.<Message>emptyList(), QueryCompletionInfo.EXCEPTION);
+        } else if (cause instanceof DataAccessException) {
+            LOGGER.error("Caught exception on channel id = {}, from = {} for query = {}\n{}",
+                    channel.getId(),
+                    ChannelUtil.getRemoteAddress(channel),
+                    query,
+                    cause.toString());
```

Comment:
Why not adding the cause.toString() always? Are the ones of the 'else' not interesting?

NOTE: in the else is the 'cause' ignored due to less '{}' then parameters.